.circleci/config.yml

```diff
@@ -25,9 +25,6 @@ commands:
         type: string
         default: ""
     steps:
-      - setup_remote_docker:
-          version: << pipeline.parameters.docker_version >>
-          docker_layer_caching: true
```

Comment:
This is not triggered here. I believe because this block is in `commands` instead of `jobs`.